src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -66,6 +61,26 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+Following list of attributes are added by default on activity. See
```

Comment:
A lot of these attributes are only added when they are available. For example, `error.type`, `user_agent.original` etc. 

```suggestion
Following list of attributes are added by default on activity when available. See
```